src/docs/implementation/SwateManual/Docs03-Building-Blocks.md

```diff
@@ -0,0 +1,522 @@
+| Leaf                            	| PO                                 	| http://purl.obolibrary.org/obo/PO_0025034 	| 28 days after   germination 	| user-specific                      	| user-specific                            	| Mature                                	| user-specific                      	| user-specific                            	|
+| Leaf                            	| PO                                 	| http://purl.obolibrary.org/obo/PO_0025034 	| 28 days after germination   	| user-specific                      	| user-specific                            	| Mature                                	| user-specific                      	| user-specific                            	|
+| Leaf                            	| PO                                 	| http://purl.obolibrary.org/obo/PO_0025034 	| 28 days after   germination 	| user-specific                      	| user-specific                            	| Mature                                	| user-specific                      	| user-specific                            	|
+
+</p>
+</details>
+
+</p>
+</details>
+
+<br>
+
+## Protocol Columns
+
+These columns have been added to better support conversion to endpoint repositories.
+
+### Protocol Type
+
+**Definition:** The type of protocol described with the assay. This building block is handled by Swate as a [descriptive building block](#descriptive-building-blocks).
+
+**Expected Values:** **MUST** be a *Protocol Type* child term.
+
+**Examples**
+
+<details><summary>Example | Growth Protocol.</summary>
+<p>
+
+| Protocol Type	| Term Source REF (NFDI4PSO:1000161) 	| Term Accession Number (NFDI4PSO:1000161)        	|
+|------------------------------	| ------------------------------	|-------------------------------------------	|
+| growth protocol   | NFDI4PSO | http://purl.obolibrary.org/obo/NFDI4PSO_1000162	|
+
+</p>
+</details>
+<br>
+
+### Protocol REF
+
+**Definition:** The name of the protocol. By default the name is inferred by the name of the Excel worksheet. To circumvent some limitations (lenght, etc.) we also offer this column as alternative. This building block comprises only one column.
+
+**Expected Values:** **MUST** be string.
+
+## Output Building Blocks
+
+**General Definition:** Each *Annotation Table* **MUST** contain exactly one output column.
+
+**Expected Values:** as in [Source Name](#input-building-blocksource-name).
+
+**Examples:** as in [Source Name](#input-building-blocksource-name).
+
+### Sample Name
+
+**Definition:** Use this building block type to reference any samples your experimental workflow produced.
+
+### Raw Data File
+
+**Definition:** Use this building block type to reference any untransformed and unprocessed samples you have produced.
+
+### Derived Data File
+
+**Definition:** Use this building block type to reference any samples your computational workflow produced.
```

Comment:
This is a huge file, we should consider splitting it into multiple files, like you did for the installation.

Reply:
done

Reply:
needs better index though

Reply:
done